src/MutatorFactory.ts

```diff
@@ -0,0 +1,24 @@
+import {Mutator} from './api/mutant';
```

Comment:
Cool :+1:

We should move this file in the api folder, that way we can later easily seperate it from the stryker main repo into the stryker-api repo.